src/override/memcpy.cc

```diff
@@ -1,3 +1,6 @@
+// Don't complain that _snprintf_l is unsafe.  It is only unsafe in certain
+// uses and this is not one of them.
+#define _CRT_SECURE_NO_WARNINGS
```

Comment:
Can you change the code causing the warning to use `snprintf_s`? I'm concerned that switching this off might hide other issues.

Reply:
Done.